examples/tutorials/physics_benchmarking.py

```diff
@@ -0,0 +1,282 @@
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def simulate(sim, dt=1.0, get_frames=True, data={}):
+    # simulate dt seconds at 60Hz to the nearest fixed timestep
+    # print("Simulating " + str(dt) + " world seconds.")
+    observations = []
+    physics_step_times = []
+    graphics_render_times = []
+    collisions = []
+
+    start_time = sim.get_world_time()
+    while sim.get_world_time() < start_time + dt:
+        st = time.time()
+        sim.step_physics(1.0 / 60.0)
+        mt = time.time()
+        if get_frames:
+            observations.append(sim.get_sensor_observations())
+        et = time.time()
+        physics_step_times.append(mt - st)
+        graphics_render_times.append(et - mt)
+        collisions.append(sim.get_num_active_contact_points())
+
+    if "observations" in data:
+        data["observations"] += observations
+    if "physics_step_times" in data:
+        data["physics_step_times"] += physics_step_times
+    if "graphics_render_times" in data:
+        data["graphics_render_times"] += graphics_render_times
+    if "collisions" in data:
+        data["collisions"] += collisions
+
+
+# Define each test function & store in list
+
+
+def bowl_drop_test(
+    objects,
+    num_objects=30,
+    object_scale=0.5,
+    object_speed=2,
+    post_throw_settling_time=5,
+):  # take in parameters here
+    """Drops a specified number of objects into a bowl/box and returns metrics including the time to simulate each frame, render each frame, and the number of collisions each frame. """
+    # start with bowl being bounding boxes
+
+    cfg = make_configuration()
+    try:  # Got to make initialization idiot proof
+        sim.close()
+    except NameError:
+        pass
+
+    sim = habitat_sim.Simulator(cfg)
+    #place_agent(sim, [5.45, 1.8, 1.2], np.quaternion(-0.83147, 0.2, -0.55557, -0.2))
+    sim.initialize_agent(0, habitat_sim.AgentState([5.45, 1.8, 1.2], np.quaternion(-0.83147, 0.2, -0.55557, -0.2)))
+
+
+    # get the physics object attributes manager
+    obj_templates_mgr = sim.get_object_template_manager()
+    # [/initialize]
+
+    # build bowl
+    cube_handle = obj_templates_mgr.get_template_handles("cube")[0]
+
+    bowlParts = []
+    bowlIDs = []
+    for _i in range(5):
+        bowlParts += [obj_templates_mgr.get_template_by_handle(cube_handle)]
+    bowlParts[0].scale = np.array([1.0, 0.1, 1.0])
+    bowlParts[1].scale = np.array([1.0, 0.6, 0.1])
+    bowlParts[2].scale = np.array([1.0, 0.6, 0.1])
+    bowlParts[3].scale = np.array([0.1, 0.6, 1.0])
+    bowlParts[4].scale = np.array([0.1, 0.6, 1.0])
+
+    for i in range(5):
+        part_name = "bowl_part_" + str(i)
+        obj_templates_mgr.register_template(bowlParts[i], part_name)
+        bowlIDs += [sim.add_object_by_handle(part_name)]
+        sim.set_object_motion_type(habitat_sim.physics.MotionType.KINEMATIC, bowlIDs[i])
```

Comment:
This setting may not let your other objects de-activate. Perhaps set the state first and then set these objects as `MotionType::STATIC` instead.